.changeset/tough-queens-swim.md

```diff
@@ -0,0 +1,6 @@
+---
+"vue-demo-store": minor
+"@shopware-pwa/composables-next": minor
```

Comment:
composable changeset should contain exact changes to the API of composables, not just a description